resolwe_bio/processes/import_data/sam_header.yml

```diff
@@ -0,0 +1,39 @@
+# ===================
+# Import - Header SAM
+# ===================
+---
+
+- slug: upload-header-sam
+  name: Upload Header SAM file
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/legacy:latest
```

Comment:
Any reason why this (very simple) process requires the (relatively big) legacy image, while the above use rnaseq? This could probably even work with base directly as it doesn't use any tools.

Reply:
No specific reason. All the import processes seem to use this image however. I will remove docker requirements in this process and we might want to do the same for (some) other.

Reply:
Removing requirements will break this process, because the default is the old biolinux-based image (which is no longer supported). You should use explicit requirements and use `resolwebio/base:latest` instead.

Reply:
Sorry, those don't have `latest`, you should use the `ubuntu-16.04` tag.